tracer/src/Datadog.Trace.SourceGenerators/Datadog.Trace.SourceGenerators.csproj

```diff
@@ -14,6 +14,7 @@
     <PackageReference Include="System.Reflection.Emit.Lightweight" Version="4.3.0" />
   </ItemGroup>
   <ItemGroup>    
+    <Compile Include="..\..\build\_build\CodeGenerators\EquatableArray.cs" Link="Helpers\EquatableArray.cs" />    
```

Comment:
meh, I would just duplicate tbh

Reply:
Which reminds me, [I found a bug in the equatable array implementation](https://github.com/andrewlock/NetEscapades.EnumGenerators/commit/4a595b0eb4723fd4ff9f39562bee463943dc1c7f#diff-ac3ece7cae43a08ae11879c742d49572301a58006c6e9e70dd153d60eb4d2bd7) I based this on! we should fix that here too (different PR) 😅